lib/galaxy/util/__init__.py

```diff
@@ -1882,3 +1882,14 @@ def enum_values(enum_class):
     Values are in member definition order.
     """
     return [value.value for value in enum_class.__members__.values()]
+
+
+def munge_lists(listA: Any, listB: Any) -> List:
```

Comment:
I don't think we want more functions in util/__init__.py, which we have to maintain indefinitely as the public api. Apart from that `munge_lists` is not a good name and we're not re-using this anywhere.